pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -287,16 +295,36 @@ func createPipelineSpec(log logr.Logger, tool string, commitTime int64, jbsConfi
 		"\nRUN chmod +x /root/*.sh" +
 		"\nCMD [ \"/bin/bash\", \"/root/entry-script.sh\" ]"
 
+	// Konflux Containerfile
+	// TODO: While it doesn't functionally matter, ideally we should match Konflux layout (i.e. /var/workdir etc).
 	kf := "FROM " + recipe.Image +
 		"\nUSER 0" +
 		"\nWORKDIR /root" +
-		"\nRUN mkdir -p /root/project /root/software/settings /original-content/marker && microdnf install vim curl" +
+		"\nRUN mkdir -p /root/project /root/software/settings /original-content/marker" +
+		// TODO ### HACK : How to use SSL to avoid certificate problem with buildah task?
+		// "\nENV CACHE_URL=" + "http://jvm-build-workspace-artifact-cache." + jbsConfig.Namespace + ".svc.cluster.local/v2/cache/rebuild" + buildRepos + "/" + strconv.FormatInt(commitTime, 10) +
 		"\nENV JBS_DISABLE_CACHE=true" +
 		"\nCOPY .jbs/run-build.sh /root" +
 		"\nCOPY . /root/project/source/" +
-		"\nRUN /root/run-build.sh" +
-		"\nFROM scratch" +
-		"\nCOPY --from=0 /root/project/artifacts /root/artifacts"
+		"\nRUN /root/run-build.sh"
+	// TODO: This is a bit of a hack but as Ant doesn't deploy and the previous implementation relied upon using the
+	//     BuildRequestProcessorImage we need to modify the Containerfile. In future the ant-build.sh should probably
+	//     encapsulate this.
+	if tool == "ant" {
+		kf = kf +
+			"\nFROM " + buildRequestProcessorImage + " AS build-request-processor" +
+			"\nUSER 0" +
+			"\nWORKDIR /root" +
+			"\nCOPY --from=0 /root/project/ /root/project/" +
+			// Don't think we need to mess with keystore as copy-artifacts is simply calling copy commands.
+			"\nRUN /opt/jboss/container/java/run/run-java.sh " + doSubstitution(strings.Join(copyArtifactsArgs, " "), []tektonpipeline.Param{}, commitTime, buildRepos) +
+			"\nFROM scratch" +
+			"\nCOPY --from=1 /root/project/artifacts /"
+	} else {
+		kf = kf +
+			"\nFROM scratch" +
+			"\nCOPY --from=0 /root/project/artifacts /"
+	}
```

Comment:
This above block is somewhat of a hack. The original code that @dwalluck added ran in the **verify** phase - but as we can't use a shared workspace it needs to run in the build phase hence appending it to the Containerfile. However its written in Java so I have to use another `FROM`. In future we should probably look to see if we can migrate it to shell.

Reply:
Yes `CopyArtifactsCommand`. Its not a major issue and is low priority as I have 'if-blocked' it so the secondary FROM isn't used under maven/gradle